src/components/Comment/Comment.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { CommentShape } from '../shapes/CommentShape';
+
+import './Comment.scss';
+
+export const Comment = ({ name, body, email }) => (
+  <div className="item__block">
+    <h2 className="item__block--name">{name}</h2>
+    <p className="item__block--email">
```

Comment:
https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80